src/style.css

```diff
@@ -0,0 +1,83 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+body {
+  font-size: 16px;
+  font-family: Avenir-Book, Helvetica, sans-serif;
+  font-style: normal;
+}
+
+.form {
+  margin: 20px 0;
+  position: relative;
+}
+
+.search {
+  font-family: Avenir-Book, Helvetica, sans-serif;
```

Comment:
Why do you need this rule here? All nested elements have to inherit font-family from "body" I think.
Can you check it?

Reply:
Yes, but if I explicitly state it font-family: inherit